python/tvm/hybrid/__init__.py

```diff
@@ -7,5 +7,54 @@
 2. Developers can build HalideIR by writing Python code.
 """
 
-from .api import script
+from __future__ import absolute_import as _abs
+
+from .._ffi.base import decorate
+from .._ffi.function import _init_api
+from .. import _api_internal as _tvm_internal
+from ..tensor import Tensor
+
 from .parser import parse_python
+from .util import _pruned_source
+
+
+def script(pyfunc):
+    """Decorate a python function function as  hybrid script.
+
+    The hybrid function support emulation mode and parsing to
+    the internal language IR.
```

Comment:
Question is if we have some HW intrinsics in dump, how can this PR reload it?

 ```
allocate Filter_L0B[float16 * 64 * 32]
    for (idx_2d_loop, 0, 4) {
      load_cbuf_to_cb(tvm_access_ptr(0.000000h, Filter_L0B, (idx_2d_loop*512), (2048 - (idx_2d_loop*512)), 2), tvm_access_ptr(0.000000h, B_local_L1, (idx_2d_loop*512), (144 - (idx_2d_loop*512)), 1), 0, 2, 1, 0, tvm_cce_string_print("false"))
    }
```

Reply:
What do you mean by reload?

Reply:
In term of functionality, it is supposed to be, but in some case, the variable names may change.
It is some kind of pseudo-bootstrap.

Reply:
That's good, we can test and give feedback when you finish this PR